app/models/forms/gis_poster.rb

```diff
@@ -6,24 +6,23 @@ class GisPoster < GenericTischDeposit
                      :bibliographic_citation, :subject, :corpname, :attachment, :license]
   # def copy_attributes
   #  super
-  #  @tufts_pdf.description = ["#{description}  Submitted in partial fulfillment of the grant requirement of the Tufts Summer Scholars Program."]
+  #  @tufts_pdXFf.description = ["#{description}  Submitted in partial fulfillment of the grant requirement of the Tufts Summer Scholars Program."]
```

Comment:
Typo?

Reply:
Yeah, I must have been trying in vain to open a file... `C-x C-f`